### ISSUE TITLE
Test project gives DllNotFoundException

### ISSUE THREAD

**Issue by equin0x80** [lioncash/SidplaySharp#1]
In Visual Studio 2017, I compile and run your test project SidplaySharpExample. It throws an exception in the SidReader constructor, when calling SidplayNative.InitializePlayer:

System.DllNotFoundException: 'Unable to load DLL 'libsidplay': The specified module could not be found. (Exception from HRESULT: 0x8007007E)'

However, libsidplay is present in the same folder as the compiled executable. What could be wrong?

**Comment by DomeAlessi** [comment]
Same thing here, the example is not working correctly

**Comment by Kashouryo** [comment]
The DLL provided was compiled to use with MSVCR110D.DLL which is a Visual C++ debug DLL. @lioncash Can you please update the libsidplay.dll to a Release version? Since it's pretty rare to encounter a computer with such an old VC SDK installed.

**Comment by lioncash** [comment]
This was a tiny project I threw together when I was in my senior year of high school and haven't looked at in close to 8/9 years. Given the large amount of time, it'd probably be better to look at [libsidplayfp](https://sourceforge.net/projects/sidplay-residfp/) and make a native interface around it instead, given the large number of emulation improvements that have been made in that timespan.

**Comment by lioncash** [comment]
Ideally, at the time, I'd have liked to have just included the source files of libsidplayfp as its own project in the solution, so that providing a binary wasn't necessary at all, and a user could just compile it using any future version of MSVC easily. However my high schooler self at the time didn't know how to integrate autoconf/makefile libraries in a way that wasn't a pain, but still worked with MSVC

**Comment by DomeAlessi** [comment]
Yeah that's a bit tricky to put together those SID libs and ofc it's also nice to create projects to experiment things.
This said, your project is one of the first that shows in the search, and already 3 ppl me included stumbled on it thinking it was working, so personally i spent some time trying to figure out what was it.
Perhaps if you do not have intention to fix it you could make it private? Or you can specify what are the constraints that don't make it work so new users will know it is not functional and maybe look for something else?

**Comment by lioncash** [comment]
Sure, I've updated the readme to indicate the repository is no longer maintained and I'll archive it to make it a little more obvious. Sorry about the trouble
